metrics/storage/webtooling.sh

```diff
@@ -53,6 +63,22 @@ Usage: $0 <count>
 EOF
 }
 
+# script used to launch the workload
+create_start_script() {
+	rm -rf "$src_dir/$start_script"
```

Comment:
I think you can use `rm -f $CMD`, right?

Reply:
@devimc thank you